library/ssl_tls.c

```diff
@@ -607,6 +607,28 @@ static void ssl_calc_finished_tls_sha384( mbedtls_ssl_context *, unsigned char *
 #endif
 #endif /* MBEDTLS_SSL_PROTO_TLS1_2 */
 
+#if defined(MBEDTLS_KEY_EXCHANGE__SOME__PSK_ENABLED) && \
+    defined(MBEDTLS_USE_PSA_CRYPTO)
+static int ssl_use_opaque_psk( mbedtls_ssl_context const *ssl )
```

Comment:
Minor: I'd prefer the name `ssl_psk_is_opaque()`. I find "use" slightly weird for a boolean function.

Reply:
The name `ssl_psk_is_opaque()` doesn't reflect very well that there's not a single PSK in the SSL context, but one has to consider both the static and the dynamic PSK configuration to see which PSK to actually use. I'd be inclined to stick with the present name, but if you insist I'll change it.

Reply:
I won't insist.